### PR TITLE
Only english comments

### PR DESCRIPTION
**Pull request by PyTechMike**

### CODE REVIEW

File: src/check-comments.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+function checkComments (context) {
```
Comment (constgen) [inline src/check-comments.js]: This name doesn't describe what this function do.

Also it is not obvious why it is in `src`

File: src/rules/only-english-comments/only-english-comments-functions.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+let checkComments = require('../../check-comments');
+
+function onlyEnglishComments (context) {
```
Comment (constgen) [inline src/rules/only-english-comments/only-english-comments-functions.js]: This function doesn't require a separate module

File: test/only-english-comments.test.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+		},
+		{
+			code: '// !@#$%^&(){}[]+-_?"|'
+		},
+		{
+			code: `
+				// normal comment 1
+				// normal comment 2
+			`
+		}
```
Comment (constgen) [inline test/only-english-comments.test.js]: Add positive `/* */` comments cases

File: test/only-english-comments.test.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+			errors: [{ message: 'Avoid using non latin characters in comments' }]
+		},
+		{
+			code: '// खराब टिप्पणी',
+			errors: [{ message: 'Avoid using non latin characters in comments' }]
+		},
+		{
+			code: '// ცუდი კომენტარი',
+			errors: [{ message: 'Avoid using non latin characters in comments' }]
+		}
```
Comment (constgen) [inline test/only-english-comments.test.js]: Add cases with mixed Latin and non-Latin chars

File: test/only-english-comments.test.js [hunk 5]
```diff
@@ -29,6 +29,12 @@ ruleTester.run('comments-only-in-english', rule, {
 				// normal comment 1
 				// normal comment 2
 			`
+		},
+		{
+			code: '/* */'
+		},
+		{
+			code: '// '
```
Comment (constgen) [inline test/only-english-comments.test.js]: Just empty comments are valid cases. But we need to check also that rule determines both `//` and `/* */` as code comments

Comment (PyTechMike) [reply test/only-english-comments.test.js]: We need to test if //and /**/ supported to be commented?

Comment (constgen) [reply test/only-english-comments.test.js]: It is good to have

**Comment by PyTechMike** [comment]
Guess all bug comments are fixed

**Pull request merged**